recipes/nionui-tool/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "nionui-tool" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/nion-software/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: cbb89a12b94c48cb28c83e0da8503eaf0c7d4fdcb6720a1d97f1b172f53e51f9
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  entry_points:
+    - nionui-tool = nion.nionui_tool.command:main
+  binary_relocation: False
+  ignore_prefix_files: True
+  detect_binary_files_with_prefix: False
+  missing_dso_whitelist: "**/*"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+    - cmake
+    - make  # [not win]
+    - python 3.7
+    - qt
+    - numpy
+  host:
+    - python {{ python }}
+    - pip
+    - setuptools
+    - xorg-libxfixes  # [linux]
+  run:
+    - python
+
+test:
+  commands:
+    - test -f "$PREFIX/bin/nionui-tool"  # [osx]
+    - test -f "$PREFIX/bin/Nion UI Launcher.app/Contents/MacOS/Nion UI Launcher"  # [osx]
+    - test -f "$PREFIX/bin/Nion UI Launcher.app/Contents/Resources/bootstrap.py"  # [osx]
+    - test -f "$PREFIX/bin/nionui-tool"  # [linux]
+    - test -f "$PREFIX/bin/NionUILauncher/NionUILauncher"  # [linux]
+    - test -f "$PREFIX/bin/NionUILauncher/bootstrap.py"  # [linux]
+    - if not exist "%PREFIX%\\Scripts\\nionui-tool.exe" exit 1  # [win]
+    - if not exist "%PREFIX%\\Scripts\\NionUILauncher\\NionUILauncher.exe" exit 1  # [win]
+    - if not exist "%PREFIX%\\Scripts\\NionUILauncher\\bootstrap.py" exit 1  # [win]
+
+about:
+  home: http://github.com/nion-software/{{ name }}
+  license: Apache-2.0
+  license_family: Apache
+  license_file:
+    - '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Can you use these from the source tarball instead of copying them in to the recipe?